Assets/MixedRealityToolkit/InputSystem/Focus/FocusProvider.cs

```diff
@@ -228,6 +228,9 @@ private void Awake()
 
         private void Start()
         {
+            Debug.Assert(MixedRealityManager.IsInitialized, "No Mixed Reality Manager found in the scene.  Be sure to run the Mixed Reality Configuration.");
+            Debug.Assert(InputSystem != null, "No Input System found, Did you set it up in your configuration profile?");
```

Comment:
Is this really an assertable condition? Arent we building in the flexibility for NOT using the input system?

Reply:
> Arent we building in the flexibility for NOT using the input system?

We did, but in this case this is a component in _our_ implementation of the input system.

Reply:
> Is this really an assertable condition?

And yes.

Reply:
Then is it even possible to get into this state? Sorry if I am being dense today :)

Reply:
Never. unless someone puts this component on by hand (Which shouldn't happen as we do it for them)

Reply:
Thanks for clarifying... looks good to me.